src/vue-components/word-item-panel.vue

```diff
@@ -66,28 +70,31 @@
     },
     computed: {
       itemClasses () {
-        // console.info('********************itemClasses', this.worditem.currentSession, this.worditem)
         return {
           'alpheios-wordlist-language__worditem__active': this.important,
           'alpheios-wordlist-language__worditem__current_session': this.worditem.currentSession
         }
+      },
+      lemmasList () {
+        return this.updated ? this.worditem.lemmasList : ''
       }
     },
     methods: {
       changeImportant () {
         this.$emit('changeImportant', this.worditem.targetWord, ! this.worditem.important)
-        this.important = ! this.worditem.important
+        this.important = this.worditem.important
```

Comment:
ok I see I missed that `this.important`  gets changed properly in eventChangeImportant and doesn't need to be reversed here. So do we need this line at all here?